### DISCUSSION TITLE
How to install and use JS SDK for Directus installed through Softaculous?

### DISCUSSION THREAD

**Discussion by ingsbeauty** [directus/api#2397] [Q&A]
Hello!

I'm new to Directus and I'm creating a project with Vanilla JS and Directus. I have a shared hosting with cPanel and I can't use Node.js. Before upgrading to a new type of hosting I would love to try to use Directus installed through Softaculous. Installation worked perfectly, but now I'm stuck to the part of consuming the stored data in Directus. Is there a way to install or connect to the JS SDK in this situation?

I would really appreciate your help and thanks for reading!

**Comment by benhaynes** [comment]
What version of Directus did Scriptaculous install?

**Reply by ingsbeauty** [reply]
It is 8.8.1

**Comment by ingsbeauty** [comment]
Has no one tried this method yet?
